src/window.rs

```diff
@@ -21,21 +21,21 @@ use crate::utils::AsAny;
 /// Whenever you receive an event specific to a window, this event contains a `WindowId` which you
 /// can then compare to the ids of your windows.
 #[derive(Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
-pub struct WindowId(pub(crate) platform_impl::WindowId);
+pub struct WindowId(usize);
```

Comment:
What's the reason to change to `usize`? I'd prefer to stick with `u64` since it's less likely to break than variable sized type.

Reply:
We had a small discussion on Matrix: https://matrix.to/#/!DGpLzJTRzBDTwZiogk:matrix.org/$PIiYlomDTxSSdKDaAmG_ZmXIYb-UMUXXKc-dC_9qXME?via=matrix.org&via=mozilla.org&via=trufflepig-forensics.com.

Imho, I'm in favor of switching to `usize`.
If we decide to bitpack more information into this type is a discussion we can have when we actually get there.